packages/packagers/js/src/helpers.js

```diff
@@ -33,7 +33,9 @@ if (parcelRequire == null) {
 
 export const helpers = {
   $parcel$export: `function $parcel$export(e, n, v, s) {
-  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
+  if (!(n in e)) {
```

Comment:
Should we use `hasOwnProperty` like below?

Reply:
I think neither version would work. I've added another variation of the test

```js
$parcel$export(module.exports, "foo", () => (parcelRequire("actualAsset")).foo);
$parcel$export(module.exports, "foo", () => (parcelRequire("phantomAsset")).foo); // => undefined
```
```js
$parcel$export(module.exports, "foo", () => (parcelRequire("phantomAsset")).foo); // => undefined
$parcel$export(module.exports, "foo", () => (parcelRequire("actualAsset")).foo);
```
Merely checking if there already exists such an export isn't enough...